src/compas/geometry/shapes/box.py

```diff
@@ -263,6 +265,10 @@ def diagonal(self):
     def dimensions(self):
         return [self.xsize, self.ysize, self.zsize]
 
+    def axes(self):
```

Comment:
i would prefer we don't do this. first of all, we would have to do it many other places for consistency, since many other objects now have a frame. secondly, we already have too many of these aliases/shortcuts as it is. they create unnecessary bloated APIs in my opinion...

Reply:
ok, makes sense.